packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -37,7 +37,7 @@ export interface RegistrationConfig {
   fieldConfigurations: {
     name: {
       displayMiddleName: boolean;
-      unidentifiedPatient: boolean;
+      allowUnidentifiedPatients: boolean;
```

Comment:
We shouldn't rename configuration properties in `fix`-type PRs. The idea of the (fix), (feat), etc. is to allow us to judge when releasing how much of a SemVer bump we need. Changing existing configuration property names should at least be a `feat`, since it will break existing working configurations.

Reply:
I feel the new name is more relevant to what it stands for. May I push this PR as a feat then?

Reply:
Yes! I agree it’s a much better name!